rosidl_typesupport_opensplice_cpp/resource/srv__type_support.cpp.em

```diff
@@ -146,14 +149,12 @@ public:
                "the data_values and info_seq were not obtained from this "
                "@(__dds_sample_type_prefix)@(suffix)_DataReader";
       case DDS::RETCODE_OK:
-        *taken = true;
-        return nullptr;
+        break;
```

Comment:
Any reason for changing from return to break here? It looks like it effectively makes no difference, I'm just curious.